### ISSUE TITLE
What to select when script starts asking about web server/phpmyadmin db etc?

### ISSUE THREAD

**Issue by nardev** [servisys/ispconfig_setup#240]
What to select when script starts asking about web server/phpmyadmin db etc.?

**Comment by nardev** [comment]
And yes, it fails.. not default user/pass no feedback..

**Comment by manzwebdesigns** [comment]
Hi nardev,

What are you running it on (Distribution and Version)?  Any detail you can offer will give the community more to help you with.

HTH,
Bud

**Comment by nardev** [comment]
I tried that on Debian 9..default hetzners image.

**Comment by manzwebdesigns** [comment]
Did you use the minimal install or webserver?  I believe this installer is expecting a clean install with no web server installed.

**Comment by nardev** [comment]
Yes i saw, tried even that, same issue. phpmyadmin and mysql simply has it's own flow and after you pass those steps the scripts starts executing with "one line view". It's not paging the install steps properly, it's just writing/overwriting output in same  line. I believe that thing  with phpmyadmin and mariadb disrupt proper flow of steps.

**Comment by manzwebdesigns** [comment]
Okay, I did notice that today when I used it for the first time... and I haven't been able to connect to it either with ssh or http(s).

Bud

**Comment by manzwebdesigns** [comment]
Okay, I successfully installed but this time I didn't install quota.  I am using a DigitalOcean Droplet running Ubuntu 16.04.  I used the default settings on the rest.
